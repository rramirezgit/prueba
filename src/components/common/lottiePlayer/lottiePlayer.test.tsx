import renderer from 'react-test-renderer'
import LottiePlayer from './'

describe('LottiePlayer', () => {
  test('renders correctly', () => {
    const component = renderer.create(
      <LottiePlayer animationData={{}} style={{}} id="lottie" />
    )
    expect(component.toJSON()).toMatchSnapshot()
  })
})
