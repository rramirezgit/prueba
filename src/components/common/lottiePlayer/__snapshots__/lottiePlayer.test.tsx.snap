// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LottiePlayer renders correctly 1`] = `
<div
  id="lottie"
  style={{}}
/>
`;
